services/feed/feed.go

```diff
@@ -34,6 +35,8 @@ func GetFeeds(ctx context.Context, opts activities_model.GetFeedsOptions) (activ
 	return activities_model.GetFeeds(ctx, opts)
 }
 
+const maxActionContentLength = 65535 // this is the max length of mysql text column, sqlite, postgres and mssql have a higher limit
```

Comment:
I do not think we are able to show so long text in the "activity feeds"

At most, 200 or 300 runes